core/services/ocrcommon/transmitter.go

```diff
@@ -71,11 +71,18 @@ func (t *transmitter) CreateEthTransaction(ctx context.Context, toAddress common
 		return errors.Wrap(err, "skipped OCR transmission, error getting round-robin address")
 	}
 
+	var gasLimit uint64
+	if txMeta != nil && txMeta.GasLimit != nil {
```

Comment:
Who sets this field "txMeta.GasLimit"?
Will it be added in a later PR?

Reply:
Yes exactly. The place where Functions would use this is here:
https://github.com/smartcontractkit/chainlink/blob/30b73a804dfba394180abe354569dade80a71be5/core/services/relay/evm/functions/contract_transmitter.go#L164